app/src/foss/java/chat/rocket/android/dynamiclinks/DynamicLinksForFirebase.kt

```diff
@@ -0,0 +1,17 @@
+package chat.rocket.android.dynamiclinks
+
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import javax.inject.Inject
+
+class DynamicLinksForFirebase @Inject constructor(private val context: Context) : DynamicLinks {
```

Comment:
Is it like a presenter?

Reply:
This is the default dagger implementation when it's a FOSS build.  It basically does nothing because dynamic links only work with play builds.  Did that address your question?